logger.py

```diff
@@ -14,8 +15,15 @@
 # Get logging variables
 log_level = config['LOGGING']['LOG_LEVEL']
 log_file = config['LOGGING']['LOG_FILE']
-log.basicConfig(format='%(asctime)s %(levelname)s: %(message)s',
+send_telegram = config['TELEGRAM']['SEND_MESSAGES']
+
+if(send_telegram):
+    log.basicConfig(format='%(asctime)s %(levelname)s: %(message)s',
+                handlers=[logging.FileHandler(log_file), logging.StreamHandler(), TelegramHandler()])
```

Comment:
@HebelHuber This might be a good solution too. Having them all as handlers in the logger.